modules/find-failing-cis-controls.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-cluster-configuration.adoc
+:_module-type: PROCEDURE
+[id="find-failing-cis-controls_{context}"]
+= Finding failing CIS controls
+
+[role="_abstract"]
+Similar to the *Policy Violations* sunburst chart in the *Configuration Management* view, the CIS controls widget provides information about failing CIS controls.
```

Comment:
You could provide the full name of the CIS acronym before the first usage in this module. For example:

"Similar to the *Policy Violations* sunburst chart in the *Configuration Management* view, the Center for Information Security (CIS) controls widget provides"

The same applies to other modules that use the acronym. Those modules might be used in other assemblies that do not provide the full name in preceding content.